docs/configuration/configuration.md

```diff
@@ -1754,7 +1754,14 @@ queue_config:
   [ min_backoff: <duration> | default = 30ms ]
   # Maximum retry delay.
   [ max_backoff: <duration> | default = 100ms ]
-
+# Configures the sending of series metadata to remote storage.
+# It is experimental and subject to change at any point.
```

Comment:
This is a bit misleading and redundant, as all of remote write is experimental per the API stability page. This line shouldn't really be here, and if we were to have something it'd be to indicate that this is going to mean something completely different once Rob's code replaces this.

Reply:
Brian and I discussed wording in IRC a bit. Something like `Metadata configuration will change or be removed in future releases` seems reasonable here as a specific callout.